AI/MMAI/BAI/model/NNModel.cpp

```diff
@@ -0,0 +1,585 @@
+/*
+ * NNModel.cpp, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+
+#include <onnxruntime_c_api.h>
+#include <onnxruntime_cxx_api.h>
+
+#include <algorithm>
+
+#include "StdInc.h"
```

Comment:
Please make sure that StdInc.h is the very first include in file

Reply:
Done